integration/admin-client/src/main/java/org/keycloak/admin/client/resource/RoleResource.java

```diff
@@ -103,30 +103,35 @@ public interface RoleResource {
 
     /**
      * Get role members
-     * <p/>
+     * <p>
      * Returns users that have the given role
-     *
+     * </p>
+     * <p>
+     * Note: This method may not return all users. In order to retrieve all users, use paging (see
```

Comment:
Overall I like what is being proposed here. I would, however, provide a bit more info on why all users might not be correctly returned in this javadoc.

Reply:
Updated the Javadoc to be more clear: it's just the first 100 users which are returned.